htdocs/addtolist.php

```diff
@@ -26,8 +26,12 @@
 }
 
 $newlist_name = isset($_REQUEST['newlist_name']) ? trim($_REQUEST['newlist_name']) : false;
+if ($newlist_name == _('New cache list')) {
+    $newlist_name = false;
+}
```

Comment:
Thanks for the fix. :)

So far, in lib2-based code `_()` never is used for translation, but always `$translate->t()`, which does some additional processing (see lib2/translate.class.php). In this case,`$translate->t()` will effectively do the same like `_()`, but still I am a somewhat reluctant to make an exception from the rule here. Changes may be done to $translate->t(), which may break this code in addtolist.php.

So if there are no special reasons to use `_()` here, please change it to `$translate->t()`. This function currently expects at least four parameters ...